desktop/cmp/input/TextInput.ts

```diff
@@ -134,7 +146,9 @@ const cmp = hoistCmp.factory<TextInputModel>(({model, className, ...props}, ref)
             placeholder: props.placeholder,
             rightElement:
                 props.rightElement ||
-                (props.enableClear && !props.disabled && isClearable ? clearButton() : null),
+                (props.enableClear && !props.disabled && isClearable
+                    ? clearButton({testId: getTestId(props, 'clearBtn')})
```

Comment:
We should take care to use consistent suffixes for these sub-components. We used `clear-btn` above in `Select`.  I don't have a strong opinion about it - clearBtn is more our overall style, but clear-btn tracks the CSS classnames more closely, and while testids are not classnames, they kind of live in the same world.